src/service.rs

```diff
@@ -298,10 +298,6 @@ where
 		bool,
 	) -> Result<Box<dyn ParachainConsensus<Block>>, sc_service::Error>,
 {
-	if matches!(parachain_config.role, Role::Light) {
-		return Err("Light client not supported!".into());
-	}
-
```

Comment:
Is the `Role` enum removed or why do we not check this anymore?

Reply:
`Role` got removed, yes...